server/tests/role.rs

```diff
@@ -100,18 +100,49 @@ fn sheriff_basic() {
     kit::scenario!(game in Night 1 where
```

Comment:
This test is called sheriff_basic. This is not basic at all. Move the minion, jester, and politician tests to another test case.